autodeploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+# There are two npm versions used in travis (.travis.yml)
+# and only one deploy is needed 
+if [ "$TRAVIS_NODE_VERSION" != "5" ]; then
```

Comment:
In #139 I propose testing on 6 **instead** of 5, so maybe we should use 4 (LTS) here?